frontend/public/components/modals/error-modal.jsx

```diff
@@ -1,11 +1,21 @@
 import * as React from 'react';
+import * as classNames from 'classnames';
 import { ActionGroup, Button } from '@patternfly/react-core';
+import { RedExclamationCircleIcon } from '@console/shared';
 
 import { createModalLauncher, ModalTitle, ModalBody, ModalFooter } from '../factory/modal';
 
-export const ModalErrorContent = ({ error, title = 'Error', cancel = undefined }) => (
+export const ModalErrorContent = ({
+  error,
+  title = 'Error',
+  cancel = undefined,
+  showIcon = false,
+}) => (
   <div className="modal-content">
-    <ModalTitle>{title}</ModalTitle>
+    <ModalTitle className={classNames('modal-header', { 'co-error-modal__icon-header': showIcon })}>
+      {showIcon && <RedExclamationCircleIcon className="co-error-modal__icon" />}
+      {title}
+    </ModalTitle>
```

Comment:
Maybe we should include the error icon always?
Thoughts @benjaminapetersen @spadgett ?

Reply:
I've removed the changes to add the error icon. That can be determined at a later date.

Reply:
> Maybe we should include the error icon always?
> Thoughts @benjaminapetersen @spadgett ?

I say, "yes."  Especially in light of https://github.com/openshift/console/pull/4259#issuecomment-583746495

Reply:
I'm inclined to agree, an error that is important enough to be visible to a user prob should have the icon and be treated consistently.